distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array_Proxy.enso

```diff
@@ -12,8 +14,10 @@ import project.Errors.Illegal_Argument.Illegal_Argument
    type returned from polyglot calls to other languages. The most common
    use-case for it is to pass it to `Vector.from_polyglot_array` to create a
    vector backed by such custom storage.
-@Builtin_Type
 type Array_Proxy
+    ## PRIVATE
+    Value ignore
```

Comment:
What is the reason for this ignored field?

Reply:
Removing the `Value ignore` constructor turns `Array_Proxy` into a singleton and fails:
```
[error] Test org.enso.interpreter.test.MetaIsATest.typesAreNotInstancesOfThemselves failed: java.lang.AssertionError: 
[error] Type Array_Proxy shall not be instance of itself expected:
[error]     at org.enso.interpreter.test.MetaIsATest.typesAreNotInstancesOfThemselves(MetaIsATest.java:238)
```
I decided to add a constructor to _"silent"_ this tests.

The proper fix is to remove the `type Array_Proxy` altogether, I think. It is not a `type` anyway. It is just a place for the `new` and `from_proxy_object` methods. However I happily leave this for @jdunkerley, @radeusgd and for another pull request.